mobile/src/helpers/object/sort-by-dates/sort-by-dates.helper.ts

```diff
@@ -0,0 +1,20 @@
+import { compareAsc, compareDesc } from 'date-fns';
+
+type sortingType = 'desc' | 'asc';
+
+interface ShouldHaveCreatedAtProp {
+  createdAt: string;
+}
```

Comment:
I think it would be better not to link by hardcoded field, so pass this one into sortByDate helper with type keyof T

Reply:
Do you mean, we should pass it like the third prop?

Reply:
Yep, but between items and type(and default value for type like `type = 'asc'`)

Reply:
![Screenshot 2022-09-12 174543](https://user-images.githubusercontent.com/89215185/189684688-e85e88fe-2cbf-4184-bb73-c658909187f0.png)

but now I have the warning)

Reply:
Wrap key and type to object(key => sortingKey - clearly to understand)